README.md

```diff
@@ -1,9 +1,8 @@
-# Popper
-
+![popper logo](docs/figures/rsz_popper_logo.png)
+#
```

Comment:
Can we reduce the size of the image and/or center it? Also, having the badges next to the title of the project (like the [hashicorp](https://github.com/hashicorp) projects), and then the logo something like:

# Popper [![Build Status](https://travis-ci.org/systemslab/popper.svg?branch=master)](https://travis-ci.org/systemslab/popper) [![Join the chat at https://gitter.im/systemslab/popper](https://badges.gitter.im/systemslab/popper.svg)](https://gitter.im/falsifiable-us/popper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)  [![PyPI version](https://badge.fury.io/py/popper.svg)](https://badge.fury.io/py/popper) [![GitHub license](https://img.shields.io/github/license/systemslab/popper.svg)](https://github.com/systemslab/popper/blob/master/LICENSE)[![Popper Status](http://badges.falsifiable.us/systemslab/popper)](http://popper.rtfd.io/en/latest/sections/badge_server.html)

<p align="center" style="text-align:center;">
  <img src="https://github.com/systemslab/popper/raw/master/docs/figures/popper_logo.png" width="200" />
</p>

Popper is a convention and CLI tool for generating reproducible papers...

Reply:
It should be okay now @ivotron